webapp-django/crashstats/api/tests/test_views.py

```diff
@@ -203,7 +203,7 @@ def test_ProcessedCrash(self):
         url = reverse("api:model_wrapper", args=("ProcessedCrash",))
         response = self.client.get(url)
         assert response.status_code == 400
-        assert response["Content-Type"] == "application/json; charset=UTF-8"
```

Comment:
I changed all the error responses so they use `JsonResponse` and thus the content-type changed.